Unit test (`test_sampling/test_continuous_sampling.py`) fails on mac OS

### Subject of the issue
Failing unit test on mac OS, stemming from differences in floating-point approximation. 

### Your environment
* pgmpy version: `dev` branch
* Python version: 3.7.7
* Operating System: OSX

### Steps to reproduce
Run tests with `pytest` on Mac OSX

### Expected behaviour
Unit test passes

### Actual behaviour
Unit test fails with `AssertionError: False is not true`

--- a/pgmpy/tests/test_sampling/test_continuous_sampling.py
+++ b/pgmpy/tests/test_sampling/test_continuous_sampling.py
@@ -210,7 +210,7 @@ class TestNUTSInference(unittest.TestCase):
 
         np.random.seed(921312312)
         samples = self.nuts_sampler.generate_sample(
-            initial_pos=[0.2, 0.4, 2.2], num_adapt=10000, num_samples=10000
+            initial_pos=[0.2, 0.4, 2.2], num_adapt=10000, num_samples=30000
         )
         samples_array = np.array([sample for sample in samples])
         sample_covariance = np.cov(samples_array.T)

